### DISCUSSION TITLE
Question about changing request timeout

### DISCUSSION THREAD

**Discussion by john-brunelle** [airbnb/lottie-ios#1731] [General]
This is just a question. Is there anyway to modify the request timeout for the URLSession  in the Animation class downloader? I would like to set this to 10 seconds instead of the default 60 seconds for the timeoutIntervalForRequest property.

Any help appreciated thanks!

**Comment by calda** [comment]
This doesn't appear to be possible with the current API. We should probably tweak it so the `URLSession` can be customized, e.g. by adding a `session: URLSession = .shared` field that allows you to pass in a customized session with your preferred timeout. PRs are welcome, if you have any interest in making this change.

**Reply by john-brunelle** [reply]
Yes I do. This would be very helpful to us and its a simple change. I added the PR. Thanks!
